cmd/qat_plugin/README.md

```diff
@@ -119,6 +119,30 @@ $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/
 > socket creation and kubelet registration. Furthermore, the deployments `securityContext` must
 > be configured with appropriate `runAsUser/runAsGroup`.
 
+#### Automatic Provisioning
+
+There's a sample [qat initcontainer](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/build/docker/intel-qat-initcontainer.Dockerfile). Basically, the script running inside the initcontainer enables QAT VFs regardless of the type of devices. In case of 4xxx and 401xx devices, it sets [cfg_services](https://github.com/torvalds/linux/blob/42e66b1cc3a070671001f8a1e933a80818a192bf/Documentation/ABI/testing/sysfs-driver-qat) that reports the configured services (crypto services or compression services) of the QAT device.
```

Comment:
can you check the description matches with the current implementation: by default it only enables SR-IOV. With compatible devices, if `qat.conf`  is made available through a configmap, it also reconfigures the services.

Reply:
I hope my change was satisfying enough to you.!